libraries/vircadia-client/tests/version.cpp

```diff
@@ -0,0 +1,28 @@
+#include <../src/version.h>
+
+#include <catch2/catch.hpp>
+#include <string>
+
+using namespace std::literals;
+
+TEST_CASE( "Sanity checks for client API version information.", "[client-api-version]" ) {
```

Comment:
```suggestion
TEST_CASE("Sanity checks for client API version information.", "[client-api-version]") {
```
Unnecessary whitespace here and in REQUIRE statements below.